beta/nncf/tensorflow/quantization/utils.py

```diff
@@ -0,0 +1,41 @@
+"""
+ Copyright (c) 2021 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from typing import List
+
+import tensorflow as tf
+
+from beta.nncf.tensorflow.layers.wrapper import NNCFWrapper
+
+
+def apply_saturation_issue_fix(model: tf.keras.Model, op_names: List[str]) -> None:
+    if not isinstance(model, tf.keras.Model):
+        raise ValueError(
+            f'Expected model to be a `tf.keras.Model` instance but got: {type(model)}')
+
+    for layer in model.layers:
+        if isinstance(layer, NNCFWrapper):
+            for weight_attr, ops in layer.weights_attr_ops.items():
+                for op_name in ops:
+                    if op_name in op_names:
+                        apply_saturation_fix_to_layer(layer, weight_attr, op_name)
```

Comment:
Why we can't use the `collect_wrapped_layers()` method instead of `isinstance(layer, NNCFWrapper)`?
If we can, I suggest looking at `get_nncf_operations()` method. Seems like it does what you need.

Reply:
Please check